### PR TITLE
Optimize v5 parser

### PR DESCRIPTION
**Pull request by tagomoris**
I optimized 30x faster than original implementation.
It can be confirmed by this bench tool. https://github.com/mshindo/NetFlow-Generator

### CODE REVIEW

File: lib/fluent/plugin/netflow_records.rb [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+          self.label
+        end
+      end
+
+      class Header < BinData::Record
+        endian :big
+        uint16 :version
+      end
+
+      class Netflow5PDU < BinData::Record
```
Comment (repeatedly) [inline lib/fluent/plugin/netflow_records.rb]: Can we remove this definition?
New parser doesn't use this class.

File: test/test_parser_netflow.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    parser = Fluent::TextParser::NetflowParser.new
+    parser.configure(conf || Fluent::Config::Element.new('ROOT', '', {}, []))
+    parser
+  end
+
+  def test_configure
+    assert_nothing_raised do
+      parser = create_parser
+    end
+  end
```
Comment (repeatedly) [inline test/test_parser_netflow.rb]: Can you get actual value from NetFlow-Generator?
Adding one test for actual example is good for future change.

File: lib/fluent/plugin/netflow_records.rb (hunk 1 above)
Comment (tagomoris) [inline lib/fluent/plugin/netflow_records.rb]: I'll have a plan to add tests, using this definitions to generate bytes representation.

Comment (tagomoris) [inline lib/fluent/plugin/netflow_records.rb]: I might move this definition into test files or so... anyway, I will need this file anyway.

File: test/test_parser_netflow.rb (hunk 2 above)
Comment (tagomoris) [inline test/test_parser_netflow.rb]: I can... but not right now.

**Comment by tagomoris** [comment]
Done for v5 protocol parser optimization.

**Comment by repeatedly** [comment]
Could you add https://github.com/fluent/fluent-logger-ruby/blob/master/.travis.yml#L9 line to `.travis.yml` for avoding bundler version issue?

**Comment by tagomoris** [comment]
@repeatedly done.

**Comment by repeatedly** [comment]
Awesome patch!

**Comment by tagomoris** [comment]
Could you release newer version?

**Comment by repeatedly** [comment]
please wait a minute

**Comment by repeatedly** [comment]
Release new verion and add you to collabolators.

**Comment by tagomoris** [comment]
Cool. Thanks.

**Pull request merged**